package.json

```diff
@@ -19,6 +19,7 @@
     "publish:hotfix": "npx lerna publish --dist-tag latest --no-verify-access --conventional-graduate bump patch",
     "format": "npx prettier -w packages/**/src/**/*.ts",
```

Comment:
If I run this script I do not get a diff. Turns out the single quotes around the pattern match are critical. I wouldn't expect npm run scripts to do glob argument expansion but I guess they do.

Reply:
oh wow how'd I miss that there was already a format script. Updated the PR to remove the existing non-working version